Cython/Compiler/Code.py

```diff
@@ -2060,131 +2060,97 @@ def entry_as_pyobject(self, entry):
         type = entry.type
         if (not entry.is_self_arg and not entry.type.is_complete()
             or entry.type.is_extension_type):
-            return "(PyObject *)" + entry.cname
+            return "(PyObject *)" + entry.cname # FIXME need to implement this
```

Comment:
?

Reply:
Apart from one call in `ModuleNode.py` this function's now basically unused. Instead I use `PyrexTypes.PyObjectType.as_pyobject()`. However, that's missing the the `not entry.is_self_arg` because I couldn't find a good way of passing it in. The comment was a reminder to look at that again.

Since it's a fairly minor bit of generated code-tidiness (it only avoids an extra cast, and I couldn't work out exactly what causes `is_self_arg` to be set anyway) I think it's simplest just to leave it out (and delete the comment).